### ISSUE TITLE
Weather conditions

### ISSUE THREAD

**Issue by i3laze** [bbernstein/homebridge-snowswitch#2]
Thank you for the great appliance - I use it with my ice melting roof system.

Personally, I’d like you to allow extra weather conditions to be added as variable.
When the “DarkSky” reports “light snow” for my location it’s actually pretty decent here and requires melting, but the plugin doesn’t turn on.

**UPD:** I’ve investigated Details pane on their website - seems “light snow” stands for precipitation probability 11% to 25% and “snow” for >=26%, so it’s this line 46 to be upgraded:
`&& forecast.precipProbability > 0.5;`

**Comment by bbernstein** [comment]
Thanks for the input. This is my first winter using DarkSky so still learning how to interpret this. I haven't had enough snow here in Boston to get much real-life testing in, so I appreciate it. Perhaps I'll make a config option for that or get more optimistic/pessimistic about the snow and look for probability >0.1 instead of 0.5. Digging in.

**Comment by bbernstein** [comment]
I just created a branch with a change to include in config:
precipProbabilityMin
(number from 0.0 to 1.0 of probability of precipitation to consider it "snowy").
branch: [add-probability](https://github.com/bbernstein/homebridge-snowswitch/tree/add-probability)
I'm out of town starting tomorrow afternoon, so I may merge this to master tomorrow. Default is to work the same as it does currently. I cleaned a little code while there too.
